.eslintrc.yml

```diff
@@ -20,13 +20,19 @@ rules:
   comma-dangle: [2, always-multiline]
   eqeqeq: [2, "allow-null"]
   no-shadow: 1
+  no-unused-vars: 1
```

Comment:
The severities will be increased in follow ups.

Reply:
similarly, all test-specific changes should go in an "overrides", not apply to the whole repo

Reply:
ditto